src/test/java/Homework17.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework17 extends BaseTest {
+@Test
+    public void addSongToPlaylist() throws InterruptedException {
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/224629130-9f363e2a-b9db-4c50-b2a4-b3ab5572725a.png)

Reply:
yah i did that already